server/plugin/subscriptions.go

```diff
@@ -203,6 +231,42 @@ func (p *Plugin) Subscribe(ctx context.Context, githubClient *github.Client, use
 		Flags:      flags,
 	}
 
+	subscribedProjects := sub.ProjectTitles()
+	if len(subscribedProjects) > 0 {
+		userInfo, apiErr := p.getGitHubUserInfo(userID)
+		if apiErr != nil {
+			return errors.Errorf("Failed to get GitHub user info for userID:%s", userID)
+		}
+
+		graphqlClient := p.graphQLConnect(userInfo)
+		if graphqlClient == nil {
+			return errors.Errorf("Failed to get graphql client")
+		}
+
+		// currently only queries for organizational projects not user repo projects
+		// org level projects and repo level projects are both included, both can be used in issue/pr assignment
```

Comment:
Do you mind expanding on this comment? By "organizational projects", is what is meant that org-level and repo-level projects belonging to an org are shown? Then only user-level repo projects are excluded?

```suggestion
		// currently only queries for organizational projects, not user repo projects
		// org level projects and repo level projects are both included, both can be used in issue/pr assignment
```